compiler/rustc_feature/src/active.rs

```diff
@@ -435,9 +435,6 @@ declare_features! (
     (active, exhaustive_patterns, "1.13.0", Some(51085), None),
     /// Allows explicit tail calls via `become` expression.
     (incomplete, explicit_tail_calls, "1.72.0", Some(112788), None),
-    /// Allows using `efiapi`, `sysv64` and `win64` as calling convention
-    /// for functions with varargs.
-    (active, extended_varargs_abi_support, "1.65.0", Some(100189), None),
```

Comment:
we don't delete features, we instead move them to `accepted.rs`

Reply:
Oh, right, I missed the part where I paste this back, thanks